apps/unified_connector/tests/test_source.py

```diff
@@ -35,7 +35,7 @@ def _connector_response_check(self, source_type, response_mock):
         mock_data = ConnectorSourceResponseMock(source_type)
         response_mock.side_effect = mock_data.get_content_side_effect
         source = ConnectorSourceFactory.create(unified_connector=self.uc, source=source_type, params=mock_data.params)
-        leads_result, count = source.source_fetcher().get_leads(source.params)
+        leads_result, count = source.source_fetcher().get_leads(source.params, None)
```

Comment:
might create issue while unpacking if get_leads(....) is None.

Reply:
The function always returns [array of leads, count]